web/src/lib/utils/people-utils.ts

```diff
@@ -69,3 +71,52 @@ export const getBoundingBox = (
   }
   return boxes;
 };
+
+export const zoomImageToBase64 = async (face: AssetFaceResponseDto, assetId: string): Promise<string | null> => {
```

Comment:
The current code has different behavior for images vs videos while this doesn't. Is there a particular reason for that?

Reply:
I moved the method to a utils files, because it's used in two different files. The method used the open image in the photoeditor for the thumbnail generation if the asset was an image. If the method is in the utils file, I can't be sure that it will be called only when the photoeditor is open. I think this dependency would be bad, so I simplified it to always query the thumbnail from the server.

Reply:
That dependency exists because it avoids to do a network request for a ressource that has already been fetched.

Reply:
Ok, I added the dependency again.